### PR TITLE
PUBDEV-6567: add support for predict path to leaf

### PR DESCRIPTION
**Pull request by honzasterba**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -32,17 +32,17 @@ subprojects {
     }
 
     compileJava {
-        sourceCompatibility = JavaVersion.VERSION_1_7
-        targetCompatibility = JavaVersion.VERSION_1_7
+        sourceCompatibility = JavaVersion.VERSION_1_8
```
Comment (michal-raska) [inline build.gradle]: Could you please separate the Java version upgrade into separate PR?

**Review by michal-raska — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (honzasterba) [reply build.gradle]: done
